### ISSUE TITLE
Aspect Ratio: Incorrect import

### ISSUE THREAD

**Issue by phenomen** [huntabyte/shadcn-svelte#622]
### Change Type

Correction

### Proposed Changes

Page: https://www.shadcn-svelte.com/docs/components/aspect-ratio

Missing `$` in the lib import alias
```html
<script lang="ts">
  import { AspectRatio } from "lib/components/ui/aspect-ratio";
</script>
```

**Comment by huntabyte** [comment]
Thanks for pointing it out!

**Comment by huntabyte** [comment]
Closed by #623
